doc/source/ray-air/doc_code/air_key_concepts.py

```diff
@@ -75,10 +75,8 @@
 batch_predictor = BatchPredictor.from_checkpoint(result.checkpoint, XGBoostPredictor)
 
 # Bulk batch prediction.
-predicted_labels = (
-    batch_predictor.predict(test_dataset)
-    .map_batches(lambda df: (df > 0.5).astype(int), batch_format="pandas")
-    .to_pandas(limit=float("inf"))
+predicted_labels = batch_predictor.predict(test_dataset).map_batches(
```

Comment:
How about we drop the map_batches and then call `predicted_labels.show()` on the next line? We shouldn't complicate examples.